### PR TITLE
Update instructions for the Android example app

### PR DESCRIPTION
**Pull request by CNugteren**
This updates the instructions for the TFLite android example app in various ways:

1. Clarifies a few minor things in the text.
2. Adds a patch file with all code-changes required in one go.
3. Applies some of the suggestions from [this comment](https://github.com/larq/compute-engine/issues/602#issuecomment-775962340) to make the dependencies-change work again.
4. Disables XNNPACK to resolve the issues [encountered here](https://github.com/larq/compute-engine/issues/602).
5. Adds instructions on how to select the right API as also [described here](https://github.com/larq/compute-engine/issues/636).
6. Replaces Bintray release support with manual downloading from the GitHub release page.

This was all tested together and I managed to classify a banana with my phone:
banana_classification

Review tip: enable `Hide whitespace changes` to get a nicer diff.

### CODE REVIEW

File: docs/compute-engine/quickstart_android.md [hunk 1]
```diff
@@ -19,39 +19,34 @@ To get started with TensorFlow Lite on Android, we recommend to carefully read t
...
-        }
-    }
-
-    dependencies {
-        implementation 'org.larq:lce-lite:0.4.0'
-    }
-    ```
+=== "Use the LCE Lite AAR from the GitHub release assets"
+    To use LCE Lite AAR in the android app, we recommend downloading the
+    [LCE package released on GitHub under 'assets'](https://github.com/larq/compute-engine/releases) to your local machine. Then follow the instructions below as if you've built the AAR locally.
```
Comment (lgeiger) [inline docs/compute-engine/quickstart_android.md:49]: It's probably easier to find if we directly link to the release:
```suggestion
    [LCE package released on GitHub under 'assets'](https://github.com/larq/compute-engine/releases/latest) to your local machine. Then follow the instructions below as if you've built the AAR locally.
```

File: docs/compute-engine/quickstart_android.md [hunk 2]
```diff
@@ -19,39 +19,34 @@ To get started with TensorFlow Lite on Android, we recommend to carefully read t
...
+The demo TensorFlow Lite Android image classification app supports two APIs: 'support' and 'task'.
+For use with Larq, we need to select the non-default support API. The two APIs are defined under
+`flavorDimensions` in `app/build.gradle`.
+
+In Android Studio, the easiest is to go to `Build` -> `Select Build Variant` and select for the four
+options: `supportDebug`, `debug`, `debug`, and `debug`. Then make sure you do a Gradle-sync before building.
+
+### 3. Add LCE-compatible TensorFlow Lite AAR to the project ###
+To add the LCE Lite AAR to your android project, you can either use the
+pre-built [LCE Lite AAR](https://github.com/larq/compute-engine/releases) (under 'assets')
```
Comment (lgeiger) [inline docs/compute-engine/quickstart_android.md:43]:
```suggestion
pre-built [LCE Lite AAR](https://github.com/larq/compute-engine/releases/latest) (under 'assets')
```

**Review by lgeiger — COMMENTED** [review]
Very nice! Thanks for updating the docs 🎉 

Do you know why XNNPack doesn't seem to work on Android? If I recall correctly it used to work in the benchmark binary. In any case this is not a big restriction, so we can just recommend disabling it for now.

**Comment by CNugteren** [comment]
> Do you know why XNNPack doesn't seem to work on Android?

I didn't really investigate. The error message I got was that it needs to be installed as a dependency but it isn't. My best guess is that either the `exclude` or the removed TensorFlow-lite include has XNNPack as dependency internally, and LCE doesn't have it. We can probably explicitly re-add the dependency, but i thought just disabling it would be acceptable as well.

**Comment by lgeiger** [comment]
> We can probably explicitly re-add the dependency, but i thought just disabling it would be acceptable as well.

Sounds good, disabling is definitely acceptable here.

**Comment by Tombana** [comment]
I went through the instructions with a fresh android-studio install, it all works :+1:

**Review by Tombana — APPROVED** [review]

**Pull request merged**